collectors/hra/parser.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from .record import Record
+from datetime import date
+import logging
+import six
+logger = logging.getLogger(__name__)
+
+
+def parse_response(application, response, from_date, to_date):
```

Comment:
It looks like we don't use `from_date` and `to_date` here? It's great but than this args should be removed.